db/src/main/kotlin/edu/artic/db/models/ArticObject.kt

```diff
@@ -87,6 +87,14 @@ data class ArticObject(
             return thumbnailFullPath?.asCDNUri()
         }
 
+    /**
+     * Alias for just the artis
+     */
+    val tombstone: String?
+    get() {
+        return artistCulturePlaceDelim?.replace("|", "/r")
```

Comment:
Any special reason for `/r`? Not meant to be `\r`, is it?

Reply:
my mistake. it should be `\r`